CHANGELOG.md

```diff
@@ -7,10 +7,19 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Added `cartesi-rollups-node` Go binary as a single entrypoint to execute all Cartesi Node services
+- Added Go-related workflows to the CI
```

Comment:
I would remove the CI-related changes from the CHANGELOG because they are irrelevant to external parties. Us developers can use `git log` to track these changes.